### PR TITLE
Free Software instead of Open Source

### PR DESCRIPTION
**Pull request by freephile**
The language of the policy should use the term 'Free Software' (and free software licenses) to protect the freedoms of the American People.  See the argument at https://github.com/WhiteHouse/source-code-policy/issues/83 which clearly and succinctly describes why this is critical for equal justice under the law.

### CODE REVIEW

File: pages/Appendixa.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ description: "Appendix A: Definitions"
 
 **Open Source License**: OSS is often associated with a license that details the terms and conditions governing the intellectual property rights of the software and its associated source code. These licenses specify how a particular work may be reproduced, modified, or used as a component of a larger system or as a standalone piece of software.<sup id="fnr49"><a href="#fn49">49</a></sup>
 
-**Open Source Software (OSS)**: Software that can be freely accessed, used, changed, and shared (in modified or unmodified form) by anyone. OSS is often distributed under licenses that comply with the definition of "Open Source" provided by the Open Source Initiative ([https://opensource.org/osd](https://opensource.org/osd)).<sup id="fnr50"><a href="#fn50">50</a></sup>
+**Free Software (OSS)**: Software that can be freely accessed, used, changed, and shared (in modified or unmodified form) by anyone. OSS is often distributed under licenses that comply with the definition of "Open Source" provided by the Open Source Initiative ([https://opensource.org/osd](https://opensource.org/osd)).<sup id="fnr50"><a href="#fn50">50</a></sup>
```
Comment (petebachant) [inline pages/Appendixa.md]: Maybe this should be "Free and open source software (FOSS)". As it is, the acronym doesn't make sense.

Comment (freephile) [inline pages/Appendixa.md]: New commit replaces OSS with FLOSS and related terms/definitions/references throughout

**Comment by pikurasa** [comment]
I agree with @freephile 

"Open Source" is ambiguous and confuses the public. Many people hear "open source" and think "charitable", "hobby software", or "not for profit".

"Free Software" on the other hand, connotes that people's _**civil liberties**_ are at stake. Richard Stallman is right to have introduced this as a _**social justice**_ issue. The Free Software Foundation continues to protect and clarify this original message.

Additionally, "free software" is not intended just to defend the civil liberties of software engineers. Much to the contrary (I am a music teacher by profession, by the way), it is to protect the civil liberties of all whose lives are affected by these tools.

Please insist that the "Land of the Free" adopt the terminology that emphasizes freedom--"Free Software" ("free" as in "free speech"). Thank you!

References:
[What is Free Software?](https://www.gnu.org/philosophy/free-sw.html.en)
["Open Source" misses the point](https://www.gnu.org/philosophy/open-source-misses-the-point.html)
[Free Software and Government](https://www.gnu.org/philosophy/government-free-software.html)

**Comment by tmcw** [comment]
The linked argument in #83 doesn't mention free software and appears to discuss a different topic. Are you sure that's the right link?

**Comment by freephile** [comment]
^ @tmcw #83 talks about the Justice argument for making software 'open' (by default).  It's not the same word as 'free' and to clear up any ambiguity, I call for 'free' by default in the comments of that issue.  The justice argument is equally applicable whether you're talking 'open' or 'free'.  However, copyleft (as defined by the GPL and the Free Software Foundation) is the legal mechanism to ensure liberty whereas 'open' is merely equivalent to a vacation from slavery; not a permanent status change. We've seen over an over again how 'open source' software is converted to proprietary software.

**Comment by tmcw** [comment]
Ah, okay, so there's an assumed moral high ground wrt copyleft licenses. Tagging out, this is a dead horse since the early 90s, not interested in spending more bytes on it.

**Comment by mattbailey0** [comment]
Thank you for your input - I'm closing this pull request given that the final language has now been released. Please feel free to reopen or create a new issue or pull if you'd like to continue the conversation!

**Pull request closed without merging**